### ISSUE TITLE
No softmax activation in BertForTokenClassification

### ISSUE THREAD

**Issue by shudima** [huggingface/pytorch-pretrained-BERT#647]
The BertForTokenClassification class has the `classifier` member, which is a linear layer. 
In the `forward` function, it treats it's out as probabilities (Cross Entropy for loss) but there's no softmax. Is there a reason for that?

**Comment by thomwolf** [comment]
It's because `nn.CrossEntropyLoss` already has a Softmax integrated in the module:
https://pytorch.org/docs/stable/nn.html?highlight=crossentropy#torch.nn.CrossEntropyLoss

**Comment by shudima** [comment]
I see it now. Thanks!
